xbmc/utils/CryptThreading.cpp

```diff
@@ -66,14 +66,14 @@ static unsigned long thread_id()
 
 CryptThreadingInitializer::CryptThreadingInitializer()
 {
-  bool attemptedToSetSSLMTHook = false;
 #ifdef HAVE_OPENSSL
   // set up OpenSSL
   numlocks = CRYPTO_num_locks();
   CRYPTO_set_id_callback(thread_id);
   CRYPTO_set_locking_callback(lock_callback);
-  attemptedToSetSSLMTHook = true;
+  bool attemptedToSetSSLMTHook = true;
 #else
+  bool attemptedToSetSSLMTHook = false;
```

Comment:
[CryptThreading.cpp:69] -> [CryptThreading.cpp:75]: (performance) Variable 'attemptedToSetSSLMTHook' is reassigned a value before the old one has been used.